src/obsidian_to_org/__main__.py

```diff
@@ -44,7 +45,9 @@ def fix_markdown_comments(markdown_contents):
 
 def restore_comments(org_contents):
     """Restore the comments in org format."""
-    return "".join(line.replace(COMMENT_MARKER, "# ") for line in  org_contents.splitlines(True))
+    return "".join(
+        line.replace(COMMENT_MARKER, "# ") for line in org_contents.splitlines(True)
+    )
```

Comment:
**Question:** What's up with the formatting changes? I don't see any mention of them in the commits or in the PR description.

Reply:
My editor formats files with `black` by default.

Reply:
Thanks! I thought mine did too, but apparently not. It seems I normally rely on pre-commit checks, which this repo doesn't have yet.

It would have made the review easier if the reformat was done in a separate commit to the logical changes.